lib/streamlit/net_util.py

```diff
@@ -92,17 +90,17 @@ def get_internal_ip():
     return _internal_ip
 
 
-def _make_blocking_http_get(url, timeout=5):
+def _make_blocking_http_get(url: str, timeout: float = 5) -> Optional[str]:
```

Comment:
nit: not particularly important, but should this be written as 5.0 instead of 5 to emphasize that it's a float?

Reply:
I don't hold this opinion strongly, but I think no. The type annotation is already there to tell you the type -- the reason to use a decimal in C-like languages is to prevent coercion-to-int in statements that would result in the loss of the decimal value, but I think that's not an issue in Python. (Like, `1/3` in Python results in a float even though both values are ints.)

Reply:
(That said, there may very well be some best practice I'm not aware of in which case my answer would change to "yes definitely!")